Updated enterprise applications deletion steps (#99268)
Azure/azure-content: articles/active-directory/enterprise-users/directory-delete-howto.md

[!Warning]: Before you proceed, to verify you are connected to the tenant you intend to delete with MSOnline module, it is recommended you run the command Get-MsolDomain to ensure you are connected to the correct tenantID and onmicrosoft.com domain.

5.	Run below command to set the tenant context 

   'Connect-AzAccount -Tenant <ObjectID of the tenant you are attempting to delete>'

[!Warning]: Before proceeding, to verify you are connected to the tenant you intend to delete with Az module, it is recommended you run the command Get-AzContext to check the connected tenant ID and onmicrosoft.com domain.
 
6.	Run below command to remove any enterprise apps with no way to delete:

    'Get-AzADServicePrincipal | ForEach-Object { Remove-AzADServicePrincipal -ObjectId $_.Id }'
 
7.	Run below command to remove application/service principals

   'Get-MsolServicePrincipal | Remove-MsolServicePrincipal'

8.	Lastly, run the command to disable any blocking service principals:

    'Get-MsolServicePrincipal | Set-MsolServicePrincipal -AccountEnabled $false'

9.	Sign back into the Azure portal and remove any new admin account created in step 3.

> 10.	Retry tenant deletion from the Azure portal again.

Author: "Retry tenant deletion from the Azure portal." Remove "again"

> If you are still unable to delete the Azure AD tenant successfully, please contact Microsoft Support.

Reviewer: ```suggestion
If you're still unable to delete the Azure AD tenant successfully, contact Microsoft Support.
```
